dashboard/.gitignore

```diff
@@ -31,3 +31,6 @@ telepresence.log
 
 # Use yarn for package management
 package-lock.json
+
+# React-app proxy
+src/setupProxy.js
```

Comment:
Added the [React app proxy](https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually) to ignore.